app/routes/_gcn.circulars._archive._index/DateSelectorMenu.tsx

```diff
@@ -118,81 +132,79 @@ export function DateSelector({
             <Grid row>
               <Grid col={4} key="radio-alltime">
                 <Radio
+                  form=""
                   id="radio-alltime"
                   name="radio-date"
                   value=""
                   label="All Time"
-                  defaultChecked={true}
-                  onChange={(e) => {
-                    setInputDateGte(e.target.value)
-                  }}
+                  defaultChecked={!defaultStartDate && !defaultEndDate}
+                  onChange={radioOnChange}
                 />
               </Grid>
               {Object.entries(dateSelectorLabels).map(([value, label]) => (
                 <Grid col={4} key={`radio-${value}`}>
                   <Radio
+                    form=""
                     id={`radio-${value}`}
                     name="radio-date"
                     value={value}
                     label={label}
-                    checked={value === inputDateGte}
-                    onChange={() => {
-                      setFuzzyTime(value)
-                    }}
+                    defaultChecked={value === defaultStartDate}
+                    onChange={radioOnChange}
                   />
                 </Grid>
               ))}
               <Grid col={4}>
                 <Radio
+                  form=""
                   id="radio-custom"
                   name="radio-date"
                   value="custom"
                   label="Custom Range..."
-                  checked={showDateRange}
-                  onChange={(e) => {
-                    setShowDateRange(e.target.checked)
+                  defaultChecked={defaultShowDateRange}
+                  onChange={({ target: { checked } }) => {
+                    setShowDateRange(checked)
                   }}
                 />
               </Grid>
             </Grid>
             {showDateRange && (
               <DateRangePicker
-                startDateHint="dd/mm/yyyy"
+                startDateHint="YYYY-MM-DD"
                 startDateLabel="Start Date"
                 className="margin-bottom-2"
                 startDatePickerProps={{
+                  form: '',
                   id: 'event-date-start',
                   name: 'event-date-start',
-                  defaultValue: 'startDate',
+                  dateFormat: 'YYYY-MM-DD',
+                  defaultValue: defaultStartDate,
```

Comment:
Is there a way for the input field of the date range selector component to have the user selected date persist upon reload? If you have a custom date range selected and open the date range selector menu, the input fields are blank rather than having the selected date values. This might be another thing that needs to be sorted out in react-uswds, however.

Reply:
Fixed now.